src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -297,6 +298,74 @@ public AgentSettings LoadSettings()
             }
         }
 
+        public async Task UnconfigureAsync(CommandSettings command)
+        {
+            if (!_store.IsConfigured())
+            {
+                _term.WriteError(StringUtil.Loc("MissingSettings"));
```

Comment:
return for not configured. should we error or just print a message? if i do unconfig twice, should i expect the second unconfig error out or just no-opt.